5주차/프로그래밍과제1-1/src/main/java/service/Calculator.java

```diff
@@ -0,0 +1,26 @@
+package service;
+
+import domain.Number;
+import domain.Operator;
+import domain.Operators;
+import view.PrinterOutputView;
+
+public class Calculator {
+
+    ExceptionCatcher exceptionCatcher = new ExceptionCatcher();
+    Operators operators = new Operators();
+
+    public void calculateFormula(String[] formula) {
+
+        Number result = new Number(exceptionCatcher.catchWrongNumber(formula[0]));
+        for (int i = 1; i < formula.length; i += 2) {
+            Operator operator = new Operator(exceptionCatcher.catchWrongString(formula[i]));
+            Number number = new Number(exceptionCatcher.catchWrongNumber(formula[i + 1]));
```

Comment:
number라는 네이밍이 와닿지 않는것 같습니다
```suggestion
            Number nextArgument = new Number(exceptionCatcher.catchWrongNumber(formula[i + 1]));
```
같은 네이밍도 있습니다.

Reply:
수정 완료